string actor_id
string actor_road_id
int32 actor_lane_id
monodrive_msgs/Waypoint actor_waypoint
monodrive_msgs/ActorLane[] lanes